library/profiler_builtins/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![feature(no_core)]
+#![no_core]
 #![no_std]
 #![feature(profiler_runtime)]
```

Comment:
Maybe group all `#![feature]` together at the top of the file?

Reply:
Done